src/components/questions/CategoriesList.vue

```diff
@@ -112,32 +45,42 @@ function pushRoute(item) {
       <div
         class="col-lg-2 mb-md-4 mb-lg-0 col-xl-2 col-xxl-2 text-center text-md-start text-primary"
       >
-        <h2 class="text-center mb-4 text-md-start mb-md-0 text-primary">
-          Categories
-        </h2>
+        <h2 class="text-center text-md-start mb-0 text-primary">Questions</h2>
       </div>
+      <p class="text-center mb-2 text-md-start mb-md-0 text-secondary">
+        <font-awesome-icon
+          icon="fa-solid fa-circle-info"
+          class="text-secondary fs-6"
+        />
+        Questions based on categories click below to category name for see list
+        of questions
+      </p>
       <div
         class="col-lg-2 my-xs-4 my-lg-0 ms-lg-5 ms-xl-4 ms-xxl-0 text-center text-md-start"
       ></div>
     </div>
     <ul class="list-unstyled mt-5">
       <li
-        v-for="item in Categories()"
-        :key="item"
+        v-for="category in Categories()"
+        :key="category"
         class="border border-light mt-4 p-2 rounded-3 mx-auto shadow-sm"
-        @click="pushRoute(item)"
+        @click="pushRoute(category)"
       >
         <div class="row">
           <div class="col-9">
             <h5 class="text-secondary pt-2">
-              {{ item }}
+              {{ category }}
             </h5>
           </div>
           <div
             class="col-1 d-flex text-primary align-items-center justify-content-center rounded-5 badge bg-light"
           >
             <h4 class="m-0 pb-1 fs-5">
-              {{ questionsList.filter(obj => obj.category === item).length }}
+              {{
+                questionsList.filter(
```

Comment:
please move this to a computed. Also, "oneQuestion" looks a bit confusing, the better name might be just "question"